ui/src/StubMediaRecorder.js

```diff
@@ -0,0 +1,8 @@
+export class StubMediaRecorder {
```

Comment:
@everyone--Going forward for testing, should we make a new file for each stub?

Reply:
Yeah, create a new file for each stub. As Winnie commented above, stubs should go in `ui/jasmine_test/spec/helpers`